src/js/permutive.js

```diff
@@ -46,6 +46,8 @@ class Permutive {
 	 * @returns {(Permutive|Array<Permutive>)} - Permutive instance(s)
 	 */
 	static init(opts, el) {
+		// Permutive must not run in the editorial preview functionality
+		if( /^(.*\.)?preview\./.test(window.location.host) ){ return false;}
```

Comment:
It looks good to me. As a refinement, in the future we could replace `preview` with `preview\.ft\.com` so that it's a bit more specific to the preview page.